src/components/teasers/teaser.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  TeaserTitle,
+  TeaserTitleLarge,
+  Text,
+  Timestamp,
+  Topline,
+} from '../typography/typography';
+import { Arrow } from '../icons/arrow';
+import { theme } from '../layout/theme';
+import { Link } from '../links/links';
+
+const TeaserContainer = styled.div`
+  cursor: pointer;
+  overflow: hidden;
+
+  img {
+    transition: transform 0.2s;
+  }
+
+  &:hover {
```

Comment:
This is spaghetti architecture but it works and I'm thankful 😁
It's spaghetti because this hover definition bleeds into many different things: p, img, general color, svg/path. This cries for problems and confusion in the future and I would rather see the objects itself handle hover.

Most probably with some prop `hover` that you can pass into the receiving components (however this would work as you only receive the instance). I have no clear guide so this is some refactor for afterwards.